cgroups/fs/hugetlb.go

```diff
@@ -25,5 +40,37 @@ func (s *HugetlbGroup) Remove(d *data) error {
 }
 
 func (s *HugetlbGroup) GetStats(path string, stats *cgroups.Stats) error {
+	pageSizes, err := getHugePageSize()
```

Comment:
We can either make a New() and use that where we instantiate today: https://github.com/docker/libcontainer/blob/master/cgroups/fs/apply_raw.go#L26

Or we can probably do an init() function and save it as a global